src/main/java/org/purpurmc/purpurextras/modules/RaidTotemDropsModule.java

```diff
@@ -0,0 +1,70 @@
+package org.purpurmc.purpurextras.modules;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Raider;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDeathEvent;
+import org.bukkit.event.raid.RaidSpawnWaveEvent;
+import org.bukkit.event.raid.RaidStopEvent;
+import org.bukkit.inventory.ItemStack;
+import org.purpurmc.purpurextras.PurpurExtras;
+
+import java.util.Map;
+import java.util.SplittableRandom;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Modifies the drop rate of totems from Evokers spawned in a raid
+ */
+public class RaidTotemDropsModule implements PurpurExtrasModule, Listener {
+
+    private final SplittableRandom random;
+    private final int dropChance;
+    private final Map<UUID, Raider> raiders = new ConcurrentHashMap<>();
```

Comment:
why are we using concurrent map here?

Reply:
on the very off chance a raider dies at the same tick a raid starts 
could also just use a sync block but c'est la vie